picocom.c

```diff
@@ -1197,6 +1197,8 @@ do_command (unsigned char c)
         fd_printf(STO, "\r\n*** pulse DTR ***\r\n");
         if ( term_pulse_dtr(tty_fd) < 0 )
             fd_printf(STO, "*** FAILED\r\n");
+        else
```

Comment:
Why?

Reply:
never mind, I got it. It's ok.

Reply:
When entering this code with `dtr_up=0` it goes out of sync afterwards.
That's why I set `dtr_up=1` after successful calling `term_pulse_dtr()`.

Reply:
I'd considered to completely get rid of those both `rts_up` and `dtr_up` variables. But I think we still need it as long as `term_get_mctl()` has not been implemented for all target platforms.

Reply:
Yes I agree. It's fine as it is.